.travis.yml

```diff
@@ -87,7 +87,7 @@ matrix:
 
         - python: 3.7
           env:
-            - PIP_DEPENDENCIES="$ASTROPY_GIT $ASDF_GIT .[test]"
+            - PIP_DEPENDENCIES="-r requirements-dev.txt .[test]"
```

Comment:
You may want to remove the `ASTROPY_GIT` and `ASDF_GIT` global vars defined up top, now that they are not used.